TWLight/static/css/new-local.css

```diff
@@ -405,6 +405,16 @@ COMMON CONTENT BLOCK CSS
     text-decoration: underline;
 }
 
+.twl-secondary-btn{
```

Comment:
having just gone and cleaned up my own rtl omissions, I just realized that these also need to be added to `new-local.rtl.css`

Reply:
Good catch! Thanks for spotting that 😄